powersimdata/data_access/data_access.py

```diff
@@ -49,89 +46,58 @@ def move_to(self, file_name, to_dir, change_name_to=None):
         """
         raise NotImplementedError
 
-    def get_base_dir(self, kind, backup=False):
-        """Get path to given kind relative to instance root
-
-        :param str kind: one of {input, output, tmp}
-        :param bool backup: pass True if relative to backup root dir
-        :raises ValueError: if kind is invalid
-        :return: (*str*) -- the specified path
-        """
-        _allowed = list(_dirs.keys())
-        if kind not in _allowed:
-            raise ValueError(f"Invalid 'kind', must be one of {_allowed}")
-
-        root = self.root if not backup else self.backup_root
-        return self.join(root, *_dirs[kind])
-
-    def match_scenario_files(self, scenario_id, kind, backup=False):
+    def tmp_folder(self, scenario_id):
         """Get path matching the given kind of scenario data
```

Comment:
Description needs to be updated:  Get path to scenario temporary folder?

Reply:
Done